src/bpftrace.cpp

```diff
@@ -1338,6 +1344,34 @@ int BPFtrace::setup_output()
   return 0;
 }
 
+int BPFtrace::set_debug_output()
+{
+  debug_epollfd_ = epoll_create1(EPOLL_CLOEXEC);
+  if (debug_epollfd_ == -1)
+  {
+    LOG(ERROR) << "Failed to create debug epollfd: " << strerror(errno);
+    return -1;
+  }
+
+  int trace_pipe_fd = open("/sys/kernel/debug/tracing/trace_pipe", O_RDONLY);
```

Comment:
Use `src/debugfs.cpp` helpers? At some point we should dynamically look at mount table to find the mntpts